data/config/__init__.py

```diff
@@ -138,7 +139,11 @@ def check_time_format(self):
         if self.time_format.lower() == "24h":
             self.time_format = "%H"
         else:
-            self.time_format = "%-I"
+            # Windows strftime syntax is different than other platforms
+            if platform.system() == "Windows":
```

Comment:
This kind of string also appears in `data/scoreboard/pregame.py` and `renderers/offday.py`